content/media/a4660e7f-d4bf-4a28-8030-9dc8576b1c9a.json

```diff
@@ -0,0 +1,166 @@
+{
+    "_links": {
+        "self": {
+            "href": "http:\/\/appserver\/media\/16\/edit?_format=hal_json"
```

Comment:
`appserver`. i know thats a lando thing but should we just set them all to `localhost`?

Reply:
Yeah, I think standardizing on these is probably a good idea. Localhost works for me. It doesn't appear to have any effect on the end result but I like it to be tidy as well.